src/main/java/com/imsweb/datagenerator/utils/DistributionUtils.java

```diff
@@ -192,8 +192,8 @@ public static CityFrequencyDto getCity(String state) {
             mapping.put(1, "zip");
             mapping.put(2, "city");
             mapping.put(3, "state");
-            mapping.put(4, "longitude");
-            mapping.put(5, "latitude");
+            // mapping.put(4, "longitude");
```

Comment:
I don't fully understand what this code does by just looking at the changes, but I don't see any reason to keep commented out code instead of just removing it.  Unless you had a specific reason to do that, I would just remove that dead code.

Reply:
The data for those fields still exists in the csv files in the specified columns. I thought it would be better to leave that information in just in case we do use those fields in the future.

Reply:
Hmm.  So the CSV file has the data to assign those fields, but you said they were not used; I assume it means they were not assigned.  That's really weird.

It does sound silly to keep data around that is not used; that makes the library bigger for no gain.

I wonder if we should also remove the data from the CSV if it's not used. 

@ctmay4 do you have an opinion on this?

Reply:
I don't feel strongly. I would be fine removing the data if it not too much work.

Reply:
We are not talking about a lot of data. @caij-ims please just leave the CSV data files alone.  Just remove the commented code and let move this on. Thanks.